### DISCUSSION TITLE
Nextjs Layouts

### DISCUSSION THREAD

**Discussion by vicnz** [vercel/next.js#32962] [Help] [answered]
`NextPage.getLayout = (page) => <>{page}</>`

Is not called, rather it's ignored

Followed the Guide from the [Layouts](https://nextjs.org/docs/basic-features/layouts)  Docs, but doesn't seem to apply to my case

**Comment by mohsen1** [comment, marked as answer]
Did you also update your `_app` according to the guide? 


```ts
// pages/_app.js

export default function MyApp({ Component, pageProps }) {
  // Use the layout defined at the page level, if available
  const getLayout = Component.getLayout || ((page) => page)

  return getLayout(<Component {...pageProps} />)
}
```
